config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+bE/yIH8aiRYji/5t5JQ7fVJkZfpR9ZplwY6jlbSs3KWmz69OJE7rKZvXVZMKLu/Biq7oWLMEaAMndk7e9B2lRy2eSlJBNzx1K60e0DsTO6cJLpOvryDoUutV73x0kvj5bU43drMQY7f0XUvoikW11x6k6gZDeAUgyw8JFN/bTgrNdmRB+kMpoT5vx2Ua13bJeWN0KXaAFKWbTxHkPiAqS+nTwJTYs5gOvp8+3RhsP4Ka0CMGmmirEUkB+3PrOvkLqsLEes8XsTsaG0JfwdFmq0yfQ9omR/50LkQ76xe+6MyXr5BXu1PLh1Z1DOwop9/FIO/1ldjRcjhBgaJ9uy7NwGncauel0eXdOoJBn+0CACRXk0CIoaPtBxXeMQYJtUyOqQJjN+vzxvkHnYJ41CxdlC2Re+rqSbuWl4Cl--zQYS/H4WdNDTdz4Z--6jtXdQQjD1iBwqtQ1FWSHQ==
```

Comment:
Whats this file about?

Reply:
We use this encrypted file to store our secrets here. To store stuff here, we need to temporarily decrypt it by running `EDITOR="editor-of-choice-here --wait" bin/rails credentials:edit` before being able to add/delete secrets here. Rails uses the `config/master.key` as the encryption key to encrypt/decrypt it.

Reply:
> Rails uses the config/master.key as the encryption key to encrypt/decrypt it.

Given that your `config/master.key` is gitignored, how can I (I use myself but think another dev) can decrypt the secrets here required to run the app?

Reply:
This key can either be shared via password managers for local development or added in as an environment variable `RAILS_MASTER_KEY` during deployment.

[Question]
Is there a preferred way of sharing these keys, including `.env` keys to other developers based on your experience? I recently came across [Doppler](https://www.doppler.com/partners/product-hunt) which seems to be a good way of doing sharing keys.

Reply:
We could use a dotenv gem like [this](https://github.com/bkeepers/dotenv). And in our file, we can call that saved variable in .env as such `ENV['OUR_SECRET_KEY']`.